docs/REST API/rest-api.md

```diff
@@ -6376,32 +6377,72 @@ None.
 }
 ```
 
-### Get Apple Business Manager (ABM)
+### List Apple Business Manager (ABM) tokens
 
```

Comment:
### Dev note

Maintain support for the old `GET /abm` endpoint for backwards compatibility.

If there are multiple Apple Business Manager connections `GET /abm` returns the following error: 

"This API endpoint has been deprecated in favor of the new GET /abm_tokens API endpoint. Please use the new endpoint documented here: https://fleetdm.com/learn-more-about/apple-business-manager-api"

Reply:
Redirect for the URL in the error message is here: https://github.com/fleetdm/fleet/pull/21043/files#diff-0c6120927d3e65309562b5b15b261d3298d05fcb0ff18e5b6512ee683b7ad6b3R532

Reply:
@marko-lisica is there a specific error code we should return in the > 1 ABM token case?

Reply:
> is there a specific HTTP status code we should return in the > 1 ABM token case?

@jahzielv I'm not sure if I understand this. What do you mean with > 1 ABM token case?

Reply:
@marko-lisica Ah sorry my b! I meant this part:
> If there are multiple Apple Business Manager connections GET /abm returns the following error:
>
> "This API endpoint has been deprecated. Please use the new GET /abm_tokens API endpoint documented here: https://fleetdm.com/learn-more-about/apple-business-manager-tokens-api"

Reply:
@jahzielv I would defer a decision to you. I think 410 (Gone) makes sense.